kit/azure/organization-hierarchy/main.tf

```diff
@@ -9,10 +14,11 @@ module "policy_root" {
 
   policy_path         = "${path.module}/lib"
   management_group_id = azurerm_management_group.parent.id
-  location            = var.location
+  location            = local.default_location
 
   template_file_variables = {
-    default_location          = "${var.location}"
+    allowed_locations         = local.allowed_locations
```

Comment:
d: what do we need default_location vs. allowed_location for? this is not clear, can we name this more specifically?